src/main/java/bio/terra/common/db/DatabaseRetryUtils.java

```diff
@@ -24,24 +25,27 @@ private DatabaseRetryUtils() {}
    * @param <T> database operation class
    * @return database operation class
    * @throws InterruptedException on thread interruption
+   * @throws DataAccessException throws the last DB operation error if maxNumAttempts is exceeded.
    */
   public static <T> T executeAndRetry(
       DatabaseOperation<T> execute, Duration retrySleep, int maxNumAttempts)
```

Comment:
As written, I think `maxNumAttempts` includes the first attempt, but it's documented as max retries

Reply:
good catch, updated doc to reflect that this is # attempts, not # retries.